charts/partners/nirmata/kyverno/v2.1.2/report.yaml

```diff
@@ -0,0 +1,94 @@
+apiversion: v1
+kind: verify-report
+metadata:
+    tool:
+        verifier-version: 1.3.0
+        profile:
+            VendorType: partner
+            version: v1.0
+        chart-uri: /nirmata/kyverno/kyverno-v2.1.2.tgz
+        digest: sha256:834c6bbf1f39fdc560467389d909fac88021ce07843440660fb0c6f08a2e9b0c
+        digests:
+            chart: sha256:834c6bbf1f39fdc560467389d909fac88021ce07843440660fb0c6f08a2e9b0c
+            package: 975a4a97281fa79ee9f984366d324ffef0b049b0715c79e6439f58fa9e61b67f
+        lastCertifiedTimestamp: "2021-12-13T14:35:43.111228+00:00"
+        certifiedOpenShiftVersions: N/A
+    chart:
+        name: kyverno
+        home: https://kyverno.io/
+        sources:
+          - https://github.com/kyverno/kyverno
+        version: v2.1.2
+        description: Kubernetes Native Policy Management
+        keywords:
+          - kubernetes
+          - nirmata
+          - policy agent
+          - validating webhook
+          - admissions controller
+        maintainers:
+          - name: Nirmata
+            email: ""
+            url: https://kyverno.io/
+        icon: https://github.com/kyverno/kyverno/raw/main/img/logo.png
+        apiversion: v2
+        condition: ""
+        tags: ""
+        appversion: v1.5.1
+        deprecated: false
+        annotations: {}
+        kubeversion: '>=1.10.0-0'
+        dependencies: []
+        type: ""
+    chart-overrides: ""
+results:
+  - check: v1.0/contains-test
+    type: Mandatory
+    outcome: PASS
+    reason: Chart test files exist
+  - check: v1.0/contains-values
+    type: Mandatory
+    outcome: PASS
+    reason: Values file exist
+  - check: v1.0/not-contains-crds
+    type: Mandatory
+    outcome: PASS
+    reason: Chart does not contain CRDs
+  - check: v1.0/helm-lint
+    type: Mandatory
+    outcome: PASS
+    reason: Helm lint successful
+  - check: v1.0/chart-testing
+    type: Mandatory
+    outcome: FAIL
+    reason: |-
+        Error running process: executing helm with args "install kyverno-2mt4sk2c8t /root/.cache/chart-verifier/_nirmata_kyverno_kyverno_v2_1_2_tgz/kyverno --namespace default --wait --values /tmp/chart-testing-989758613/values.yaml": exit status 1
+        ---
+        Error: Kubernetes cluster unreachable: Get "http://localhost:8080/version?timeout=32s": dial tcp 127.0.0.1:8080: connect: connection refused
```

Comment:
This usually happens when container was not passed the kubeconfig file this is an example on how we pass that in our tests:
`docker run --rm -i \
          -e KUBECONFIG=/.kube/config \
          -v "${HOME}/.kube":/.kube \
          "quay.io/redhat-certification/chart-verifier:latest" \
          verify \
          https://dperaza4dustbit.github.io/helm-chart/psql-service-0.1.7.tgz`

Reply:
Thank you @dperaza4dustbit, it helps!